### PR TITLE
🗄️  Project Structure

### PR DESCRIPTION
**Pull request by amelieMa**

### CODE REVIEW

File: docs/project-structure.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ src/features/awesome-feature
 |
 +-- hooks       # hooks scoped to a specific feature
 |
-+-- routes      # route components for a specific feature pages
```
Comment (wab) [inline docs/project-structure.md:48]: Why do you remove the word "specific" ?

File: docs/project-structure.md [hunk 2]
```diff
@@ -56,7 +56,60 @@ src/features/awesome-feature
 +-- index.ts    # entry point for the feature, it should serve as the public API of the given feature and exports everything that should be used outside the feature
 ```
 
-A feature folder could also contain other features (if used only within the parent feature) or be kept separated, it's a matter of preference.
```
Comment (wab) [inline docs/project-structure.md]: Why removing this precision ?

File: docs/project-structure.md [hunk 3]
```diff
@@ -56,7 +56,60 @@ src/features/awesome-feature
...
+Here is a more concrete (it's wordy, but it helps to be clear and explicit. It's up to us to find names of features that allow it):
```
Comment (wab) [inline docs/project-structure.md]: not sure thats the goal os this repo is to show concrete example from our app. The repo should be enough as an example and we should not need to maintain these exemples when app will be moving...

Comment (amelieMa) [reply docs/project-structure.md]: I do not understand.
Isn't the purpose of documentation to be maintained?
I would like more information on what is wrong please

File: docs/project-structure.md (hunk 1 above)
Comment (amelieMa) [reply docs/project-structure.md:48]: If we see the [misc](https://github.com/Yousign/bulletproof/tree/master/src/features/misc) folder, you will see several pages, not relating to a specific feature, are present there.

File: docs/project-structure.md (hunk 2 above)
Comment (amelieMa) [reply docs/project-structure.md]: I understood that we wanted flat features. But here the subject is to have features in features

**Comment by amelieMa** [comment]
Do you want me to delete my PR? @wab

**Comment by amelieMa** [comment]
I need to remove lib folder. 
Please don't merge now.

**Pull request merged**